webapp_frontend/public/index.html

```diff
@@ -24,20 +24,25 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
-    <title>React App</title>
+    <title>FileFighter</title>
   </head>
-  <body>
+  <body class="h-100">
```

Comment:
<3

Reply:
I do my best to follow your instructions, your grace